addons/l10n_it_edi_sdicoop/tests/expected_xmls/IT00470550013_basis.xml

```diff
@@ -37,8 +37,7 @@
           <IdCodice>00465840031</IdCodice>
         </IdFiscaleIVA>
         <Anagrafica>
-          <Nome>Alessi</Nome>
-          <Cognome></Cognome>
```

Comment:
Is this necessary for format_alphanumeric?

Reply:
it's not necessary no, but I've explained why I did it in the commit message.
Unless it's important to keep it the same, I think it's good to change.